tests/data/cases/preview_long_strings.py

```diff
@@ -305,6 +305,15 @@ def foo():
 # Test case of an outer string' parens enclose an inner string's parens.
 call(body=("%s %s" % ((",".join(items)), suffix)))
 
+log.info(f'Skipping: {desc["db_id"]=} {desc["ms_name"]} {money=} {dte=} {pos_share=} {desc["status"]=} {desc["exposure_max"]=}')
+
+log.info(f"Skipping: {desc['db_id']=} {desc['ms_name']} {money=} {dte=} {pos_share=} {desc['status']=} {desc['exposure_max']=}")
+
+log.info(f'Skipping: {desc["db_id"]} {desc["ms_name"]} {money=} {dte=} {pos_share=} {desc["status"]} {desc["exposure_max"]}')
+
+log.info(f'Skipping: {foo("asdf")=} {desc["ms_name"]} {money=} {dte=} {pos_share=} {desc["status"]} {desc["exposure_max"]}')
+
+log.info(f'Skipping: {  longer_longer_longer_longer_longer_longer_name   [ "db_id" ]      [ "another_key" ]   =  :  .3f }')
 
```

Comment:
Add some test cases involving triple quotes, e.g.

```
% black --line-length=20 --preview -c "log.info(f'''{\"a\" == 'b'} aaaaaa aa aa aa aa aa aa aa''')"
log.info(
    f"""{"a" == 'b'} aaaaaa aa aa aa aa aa aa aa"""
)
```

Reply:
I don't think your example is valid, as f-strings are not supposed to have backslashes inside the expressions. I wonder why the `-c` option allows this.

Reply:
These backslashes are handled by the shell, not Python. (This was on a Mac; on Windows you'd probably need different quoting.)